src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/ActionRuleSurveyEmailTemplateEndpoint.java

```diff
@@ -56,6 +56,12 @@ public ActionRuleSurveyEmailTemplateEndpoint(
   public List<String> getAllowedPackCodesBySurvey(
       @RequestParam(value = "surveyId") UUID surveyId,
       @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    authUser.checkUserPermission(
+            userEmail,
+            survey.getId(),
```

Comment:
`survey` hasn't been defined or populated yet here. This could be changed to directly use the surveyId from the request params instead.